application_form/migrations/0021_application_date_of_birth_ssn_suffix.py

```diff
@@ -0,0 +1,34 @@
+# Generated by Django 2.2.24 on 2021-06-15 09:00
+
+import datetime
+import django.core.validators
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("application_form", "0020_right_of_residence_null"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="applicant",
+            name="date_of_birth",
+            field=models.DateField(
+                default=datetime.date(2021, 6, 15), verbose_name="date of birth"
```

Comment:
Interesting that it uses these dates as a default even it not specified.

Reply:
This is a new non-null field, so this default value is for possible existing rows in the database.